### PR TITLE
Multirun per command env and args

### PR DESCRIPTION
**Pull request by Globegitter**
This is in some ways a follow-up PR to #41, in the sense that we actually realised we needed to be able to set env vars for each individual command (see https://github.com/bazelbuild/rules_k8s/issues/232) and then I thought we might as well allow args for each command to be set individually as it does not always work so well to pass the same args to all commands. Let me know what you think.

### CODE REVIEW

File: multirun/def.bzl [hunk 1]
```diff
@@ -21,7 +36,8 @@ def _multirun_impl(ctx):
         default_runfiles = info.default_runfiles
         if default_runfiles != None:
             transitive_depsets.append(default_runfiles.files)
-        content.append("echo Running %s\n./%s\n" % (shell.quote(str(command.label)), shell.quote(exe.short_path)))
+        full_command = "%s ./%s %s" % (envs, shell.quote(exe.short_path), attrs)
```
Comment (ash2k) [inline multirun/def.bzl]: Environment variables may contain special characters that cannot be just put into the script directly. I'm not sure how to properly escape it. Maybe `shell.quote()` is enough.

Comment (Globegitter) [reply multirun/def.bzl]: I addressed that comment now in a simple way - tested it and it works for our internal use-case. Let me know what you think.

This right now only works if arguments are passed in via `--a=b` but not `--a b` to keep the logic simple. Also happy to try and support the second case. Also I tried using `shell.quote` without splitting the args/envs but that did not work.

**Comment by ash2k** [comment]
I think it might be much less messy to model each runnable binary as a separate rule that exposes a provider, consumable by the multirun rule. Then each of those rules can be configured individually without messy string parsing (use dict vs parsing a string). WDYT?

**Comment by ash2k** [comment]
I couldn't make the provider idea work the way I envisioned, but I did something else instead. See https://github.com/atlassian/bazel-tools/pull/45 and let me know what you think.

**Comment by ash2k** [comment]
I've merged a different implementation, closing.

**Pull request closed without merging**